src/network.rs

```diff
@@ -0,0 +1,88 @@
+use errors::*;
+use std::path::{Path,PathBuf};
+use std::fs::{remove_file, rename};
+
+use args::Args;
+use fs;
+
+const BOOTPART_CONNECTIONS_DIR: &str = "system-connections";
+const NM_CONNECTIONS_DIR: &str = "/etc/NetworkManager/system-connections/";
+
+pub fn configure(args: &Args) -> Result<()> {
+    mirror_connections(args, BOOTPART_CONNECTIONS_DIR, NM_CONNECTIONS_DIR)?;
+    Ok(())
+}
+
+fn no_comments(s: &str) -> String {
+    let mut ret = String::new();
+    for line in s.lines() {
+        if ! line.starts_with("#") {
```

Comment:
Clippy reports that `'#'` can be used instead of `"#"`. Currently the CI is broken, but the idea is to not allow anything that produces warnings when running `cargo clippy` and `cargo fmt`.